pytest_watcher/watcher.py

```diff
@@ -5,7 +5,8 @@
 import time
 from datetime import datetime, timedelta
 from pathlib import Path
-from typing import Sequence, Tuple
+from typing import Sequence
+from dataclasses import dataclass
```

Comment:
We will need to drop support for Python 3.6 in case of using `dataclasses`. But that's probably OK though, cause this version no longer supported by Python community itself